src/Uno.Foundation/Collections/ObservableVector.cs

```diff
@@ -4,9 +4,9 @@
 using System.Text;
 using Windows.Foundation.Collections;
 
-namespace Windows.UI.Xaml
+namespace Windows.Foundation.Collections
 {
-	internal class ObservableVector<T> : IObservableVector<T>
+	public class ObservableVector<T> : IObservableVector<T>
```

Comment:
Why does it need to be public ?

Reply:
So that it can be used in a different assembly (namespace `Windows.Media.Playback`, file `src/Uno.UWP/Media/Playback/MediaPlaybackList.cs`).

Please let me know if you see a different solution.